docs/3.0.0-alpha.x/advanced/hooks.md

```diff
@@ -95,7 +95,7 @@ To activate and configure your hook with custom options, you need to edit your `
 
 ## Dependencies
 
-It happens that a hook has a dependency to another one. For example, the `strapi-hook-bookshelf` has a dependency to `strapi-hook-knex`. Without it, the `strapi-hook-bookshelf` can't work correctly. It also means that the `strapi-hook-knex` hook has to be loaded before.
+It happen that a hook has a dependency to another one. For example, the `strapi-hook-bookshelf` has a dependency to `strapi-hook-knex`. Without it, the `strapi-hook-bookshelf` can't work correctly. It also means that the `strapi-hook-knex` hook has to be loaded before.
```

Comment:
It "happens" is actually correct.  Please change back.

Reply:
Please see this explanation: https://english.stackexchange.com/questions/198450/happens-or-happen